x-pack/qa/openldap-tests/src/test/java/org/elasticsearch/test/OpenLdapTests.java

```diff
@@ -257,9 +257,10 @@ public void testResolveSingleValuedAttributeFromConnection() throws Exception {
         LdapMetadataResolver resolver = new LdapMetadataResolver(config, true);
         try (LDAPConnection ldapConnection = setupOpenLdapConnection()) {
             final Map<String, Object> map = resolve(ldapConnection, resolver);
-            assertThat(map.size(), equalTo(2));
+            assertThat(map.size(), equalTo(3));
             assertThat(map.get("cn"), equalTo("Clint Barton"));
             assertThat(map.get("sn"), equalTo("Clint Barton"));
+            assertThat(map.get("mail"), equalTo("hawkeye@oldap.test.elasticsearch.com"));
         }
```

Comment:
👍 This is what I had in mind when reviewing the `LdapRealmTests` above. So maybe randomize and add the new settings in this class as well.